src/main/java/seedu/pill/Pill.java

```diff
@@ -5,34 +5,33 @@
 import seedu.pill.util.Printer;
 import seedu.pill.util.Storage;
 import seedu.pill.util.Ui;
+import seedu.pill.util.PillLogger;
 
 import seedu.pill.exceptions.PillException;
 
-import java.util.logging.Level;
 import java.util.logging.Logger;
 
 public final class Pill {
     private static final Storage storage = new Storage();
     private static ItemMap items = new ItemMap();
     private static final Ui ui = new Ui(items);
     private static Parser parser = new Parser(items, storage);
+    private static final Logger logger = PillLogger.getLogger();
```

Comment:
I like the usage of the private static final modifiers, enforcing OOP principles